integrations/lib/multiwoven/integrations/destination/http/config/spec.json

```diff
@@ -12,15 +12,10 @@
         "title": "Destination url",
         "order": 0
       },
-      "Username": {
-        "type": "string",
-        "title": "Authentication Username",
+      "headers": {
```

Comment:
How to FE knows these headers are an array of objects(key,value)?

Reply:
This has been altered based on a conversation with FE on exact format.